ci/taskcat.yml

```diff
@@ -1,7 +1,7 @@
 global:
   marketplace-ami: false
   owner: quickstart-eng@amazon.com
-  qsname: quickstart-heptio
+  qsname: quickstart-vmware
```

Comment:
I'm not familiar with the CI setup this is used for but I believe all this needs updated as well. Please confirm this wont break something I am not aware of.

Reply:
Good question, I'm not sure where the CI config lives these days, SRE should be able to point you in the right direction, though. I also know @chuckha has contributed to the CI config in the past.

Reply:
It looks like the CI config now lives in https://github.com/heptio/infra-ctrl/blob/master/infrastructure/jenkins/jobs/generate_aws_quickstart.groovy

Although the S3 bucket itself seems like it may have been created through the console

Reply:
@jessedearing would you be able to make `vmware-ci-aws-quickstart` for us? I apparently lack the credentials to see that bucket so I guess it is the test bot account that created it. I tried my dev account and the wardroom acct but neither can access that bucket at all.